style.css

```diff
@@ -282,4 +297,141 @@ body {
     font-weight: 600;
     letter-spacing: -1px;
   }
+
+  .contact {
+    background: #f9f9fa;
+    width: 320px;
+  }
+
+  footer {
+    margin: 50px 0 100px 0;
+  }
+
+  .contact div h2 {
+    font-size: 50px;
+    margin: 0;
+    margin-left: 16px;
+    margin-bottom: 20px;
+    color: #1a2236;
+    font-family: "Playfair Display", serif;
+    line-height: 50px;
+    font-weight: 400;
+  }
+
+  .container .contact div p {
+    font-size: 14px;
+    margin: 20px 16px 32px 16px;
+    color: #1a2236;
+    font-family: "Poppins", serif;
+    line-height: 21px;
+    font-weight: 400;
+  }
+
+  .container .contact form {
+    width: 288px;
+    margin-bottom: 64px;
+  }
+
+  .container .contact form ul {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    text-decoration: none;
+    gap: 8px;
+    align-items: center;
+    width: 288px;
+    margin-left: 16px;
+    padding: 0;
+    background-color: #fff;
+  }
+
+  .container .contact form ul li label {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 15px;
+    color: #7d7c84;
+    padding: 0;
+    font-family: "Poppins", serif;
+    font-style: normal;
+    margin: 4px 124px 0 16px;
+  }
+
+  .container .contact form ul li input {
+    font-family: "Poppins", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 21px;
+    color: #1a2236;
+    border: none;
+    height: 21px;
+    padding: 0;
+    margin: 4px 16px 8px 16px;
+  }
+
+  .container .contact form ul li textarea {
+    font-family: "Poppins", serif;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 21px;
+    color: #7d7c84;
+    border: none;
+    height: 21px;
+    height: 112px;
+    margin: 15px 16px;
+  }
+
+  .contact form ul li:last-child {
+    margin-top: 12px;
+    margin-bottom: 0;
+  }
+
+  .contact form ul li button:hover,
+  .contact form ul li button:focus {
+    cursor: pointer;
+    border: 1px solid #1a2236;
+    background-color: #e5e5e5;
+  }
```

Comment:
Thank you for your review @topeogunleye.
I added before (button: hover) and (button: focus).
Do I change them?
Thank you.

Reply:
Thank you for that, I Added all these changes.